# 16S rRNA Analysis Tutorials

These tutorials were created by Maxine Ty (maxine.ty@mail.utoronto.ca) 

## Background 

16S analysis is a method of microbiome analysis (distinct from shotgun metagenomics) that targets the 16S ribosomal RNA gene, as this gene is present in all prokaryotes. It features regions that are conserved among the organisms and therefore ideal for binding of universal DNA primers, as well as variable regions that distinguish organisms by sequence. These characteristics make the gene useful for analyzing microbial communities at a reduced cost compared to metagenomics. 

The dataset included for these tutorials was originally for a study that compared how different microbial-based products would affect the gut microbial composition under a pathogen challenge. Originally 158 chicken cecal content samples were used for this project; for simplicity, only 10 samples were used in this tutorial dataset. 

## Tutorials
The two tutorials under this repository (the two pages in the Wiki tab) will take you through workflows for working with 16S rRNA data. 

1. The first tutorial will guide you in processing raw 16S rRNA data with QIIME 2 and is adapted from the LangilleLab's SOP (https://github.com/LangilleLab/microbiome_helper/wiki/Amplicon-SOP-v2-(qiime2-2018.6).
2. The second tutorial will guide you in analyzing processed 16S rRNA data with various R packages. 
